wikipedia-sentences/flows/query.yml

```diff
@@ -14,5 +14,4 @@ pods:
     shards: $JINA_SHARDS
     separated_workspace: true
     polling: all
-    uses_reducing: _merge_all
```

Comment:
sorry I did not explain myself properly, the name changes, but still we will need to have slme uses after

Reply:
but the parameter is `uses_after`?

Reply:
Fixed